src/Controls/src/Core/Compatibility/Handlers/ListView/iOS/ListViewRenderer.cs

```diff
@@ -527,6 +529,16 @@ void UpdateIsRefreshing()
 				_tableViewController.UpdateIsRefreshing(refreshing);
 		}
 
+		void UpdateIsEnabled()
```

Comment:
Could do the same just overriding https://github.com/dotnet/maui/blob/5d6753110f98f5997b46e06c920eb341f8381386/src/Controls/src/Core/Compatibility/Handlers/VisualElementRenderer.cs#L205C3-L205C40

Reply:
Done